platform-api/src/main/java/io/datacater/core/info/VersionInfo.java

```diff
@@ -0,0 +1,27 @@
+package io.datacater.core.info;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public record VersionInfo(
+    @JsonProperty("version") String version,
+    @JsonProperty("build-date") String buildDate,
```

Comment:
I would also update the JSON properties of `VersionInfo` to camel case notation. This allows us to provide a consistent API experience.